src/common/form/form.mixins.scss

```diff
@@ -27,7 +27,7 @@
 @mixin form-wide() {
     .form-group {
         position: relative;
-        padding-left: $sidebar-width;
+        padding-left: $sidebar-width*1.1;
```

Comment:
"1.1" is an oddly magic number -- there is a $space-size increment, could we use that here instead?